packages/shared/src/components/BookmarkFeedLayout.tsx

```diff
@@ -169,13 +186,20 @@ export default function BookmarkFeedLayout({
           onRequestClose={() => setShowSharedBookmarks(false)}
         />
       )}
-      <div className="mb-4 laptop:hidden">
+      <div className="relative mb-4 laptop:hidden">
         <BookmarkSection
           isItemsButton={false}
           sidebarExpanded
           shouldShowLabel
           activePage=""
         />
+        {plusEntryExp && (
```

Comment:
Once above comment is fixed:
```suggestion
        {shouldShowBanner && (
```